self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryProjectsTestCase.java

```diff
@@ -0,0 +1,97 @@
+package com.selfxdsd.core.mock;
+
+import com.selfxdsd.api.*;
+import com.selfxdsd.api.storage.Storage;
+import com.selfxdsd.core.projects.UserProjects;
+import org.junit.Test;
+import java.util.ArrayList;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Unit tests for class {@link InMemoryProjects}.
+ * @author hpetrila
+ * @version $Id$
+ * @since 0.0.1
+ * @todo #16:30min Continue writing unit tests for the InMemory storage
```

Comment:
@HoreaPetrila Aici trece ``#39``, te rog Intodeauna un todo are trecut ticket-ul parinte.

De asemena, du-te te rog in ``InMemoryTestCase`` si sterge todo-ul la care ai lucrat, ca sa se inchida ticket-ul.